src/abci/code.rs

```diff
@@ -87,8 +87,8 @@ impl<'de> Deserialize<'de> for Code {
             where
                 E: serde::de::Error,
             {
-                match val.parse::<u32>() {
-                    Ok(val) => self.visit_u32(val),
+                match val.parse::<u64>() {
```

Comment:
Why did this change?

Reply:
For better readability.
This is related to a previous PR #13
In either cases `visit_u64` method is called, so thought it would be better if it is explicit in the code.